# Generated by Django 4.1.7 on 2023-03-18 00:38

import datetime
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('cryptocurrencys', '0004_alter_nfts_total_supply'),
    ]

    operations = [
        migrations.AlterField(
            model_name='cryptocurrency',
            name='ath_date',
            field=models.DateField(blank=True, null=True, validators=[django.core.validators.RegexValidator(message='Date must be in YYYY-MM-DD format', regex='^\\d{4}-\\d{2}-\\d{2}$'), django.core.validators.MaxValueValidator(limit_value=datetime.date(2023, 3, 18), message='Date cannot be in the future')]),
        ),
        migrations.AlterField(
            model_name='cryptocurrency',
            name='atl_date',
            field=models.DateField(blank=True, null=True, validators=[django.core.validators.RegexValidator(message='Date must be in YYYY-MM-DD format', regex='^\\d{4}-\\d{2}-\\d{2}$'), django.core.validators.MaxValueValidator(limit_value=datetime.date(2023, 3, 18), message='Date cannot be in the future')]),
        ),
        migrations.AlterField(
            model_name='cryptocurrency',
            name='last_updated',
            field=models.DateField(validators=[django.core.validators.RegexValidator(message='Date must be in YYYY-MM-DD format', regex='^\\d{4}-\\d{2}-\\d{2}$'), django.core.validators.MaxValueValidator(limit_value=datetime.date(2023, 3, 18), message='Date cannot be in the future')]),
        ),
    ]
